README.md

```diff
@@ -239,6 +239,68 @@ Maven:
 </dependency>
 ```
 
+Kotlin
+-----
+For all Kotlin enthusiasts, you may wonder how to use this library to configure your intents. This is perfectly compatible, with a bit of understanding of how Kotlin works, especially when it comes to annotation processing.
+
+Assuming that your project is already configured with Kotlin, update your `build.gradle` file :
+
+```groovy
+apply plugin: 'kotlin-kapt'
+
+dependencies {
+  compile 'com.f2prateek.dart:henson:(insert latest version)'
```

Comment:
you can replace with `implementation` as `compile` is deprecated

Reply:
Done and ready for merge ;)